### PR TITLE
Adding ansible vcenter population playbook

### PR DESCRIPTION
**Pull request by pdumais**

### CODE REVIEW

File: ansible/populate_vcenter.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+- hosts: localhost
+  vars:
+    nuage_auth:
+      api_username: csproot
+      api_password: csproot
+      api_enterprise: csp
+      api_url: https://vns_dev_511.lab.llama2.cloud
```
Comment (pdellaert) [inline ansible/populate_vcenter.yml]: Can you make this an internal IP (something in the 10/8, 172.16/22 or 192.168/15 range) and add the port? Or use `localhost:8443`. I don't like to expose internal dns names in the examples

File: ansible/populate_vcenter.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
```
Comment (pdellaert) [inline ansible/populate_vcenter.yml]: Can you add some documentation/comment on the goal of this example and what it accomplishes? You can look at the other examples.

File: ansible/populate_vcenter.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- hosts: localhost
+  vars:
+    nuage_auth:
+      api_username: csproot
+      api_password: csproot
+      api_enterprise: csp
+      api_url: https://vns_dev_511.lab.llama2.cloud
+      api_version: v5_0
+
+    data:
```
Comment (pdellaert) [inline ansible/populate_vcenter.yml]: I don't see this being used?

**Comment by pdumais** [comment]
ok. Added description, remove unused variable and change localhost as vsd hostname

**Comment by pdellaert** [comment]
@pdumais would it be possible to rename the file/playbook to something more specific, like `configure_vcin_hypervisors.yml` ? That represents better what the specific use case is, i think.

**Pull request merged**